notebooks/265-qrcode-monster/265-qrcode-monster.ipynb

```diff
@@ -0,0 +1,974 @@
+{
```

Comment:
<p>Please remove empty line at the beginning of the cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>removed</p>